documentation/guidance/gsuite/collaborative-inbox-forwarding.md

```diff
@@ -0,0 +1,34 @@
+# G Suite Shared Inbox Forwarding
```

Comment:
Why are we calling this Shared Inbox if it's just Google Groups?  Seems like we should stick with what people are familiar with.

Reply:
Maybe Distribution Lists?

Reply:
They are distribution lists, so why not? As long as we choose a consistent and user-friendly name, it should be fine.

Background:
- Groups configured like ours are called _collaborative inboxes_ by the official documentation <sup>[[1]](https://support.google.com/a/answer/9325194?hl=en)</sup> <sup>[[2]](https://support.google.com/a/answer/167430?hl=en)</sup>
- As you point, they are glorified [distribution lists](https://en.wikipedia.org/wiki/Distribution_list).
- We've kept calling them _shared inboxes_ for a while, but it's not a common term.